pkg/abcupdater/version_test.go

```diff
@@ -206,3 +207,43 @@ To disable notifications for this new version, set SAMPLE_APP_1_IGNORE_VERSIONS=
 		})
 	}
 }
+
+// Note: These tests rely on timing and could be flaky if breakpoints are used.
```

Comment:
I'm not super happy with my tests. Would appreciate any tips on how to make them less fragile